PHC-Android/PHC/src/main/java/phc/android/CheckoutFormFragment.java

```diff
@@ -0,0 +1,164 @@
+package phc.android;
+
+
+import android.app.Activity;
+import android.app.FragmentTransaction;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.RadioButton;
+import android.widget.RadioGroup;
+import android.widget.Toast;
+import android.widget.RadioGroup.OnCheckedChangeListener;
+import com.google.zxing.integration.android.IntentIntegrator;
+import com.google.zxing.integration.android.IntentResult;
+
+public class CheckoutFormFragment extends Fragment {
+
+    /** Holds the result of the scan. */
+    protected String mScanResult;
+    /** The textview that holds the comment. */
+    private EditText mComment;
+    /** The checkboxview holds their input for good/bad experience */
+    private int mExperience;
+    /** The checkboxview holds their input for whether they found their services */
+    private CheckBox mServices;
+    /** Field and submit button for manual code input. **/
+    protected EditText mCodeInput;
+    protected RadioGroup mRadioGroup;
```

Comment:
could ya add quick 1 line docstrings/comments for each field you added? see above for examples. 
also, you should probably just make these fields private for now - i doubt we'll be creating a subclass out of CheckoutFormFragment that'll need to inherit these protected fields.